src/routes/api/installation/api-installation-delete.ts

```diff
@@ -6,6 +6,11 @@ export const ApiInstallationDelete = async (req: Request, res: Response): Promis
 	const githubInstallationId = req.params.installationId;
 	const jiraHost = req.params.jiraHost;
 
+	//TODO: ARC-1619 Maybe need to fix this and put it into the path
+	//Not doing it now as it might break pollinator if it use this api
+	const { gitHubAppIdStr } = req.query;
+	const gitHubAppId = parseInt(gitHubAppIdStr as string) || undefined;
```

Comment:
isn't gitHubAppIdStr already a string?

Reply:
Got this 
![image](https://user-images.githubusercontent.com/105693507/185543846-1a647286-367e-405e-9acc-db31b5f7cabe.png)